Library/Formula/freeimage.rb

```diff
@@ -26,6 +26,13 @@ class Freeimage < Formula
 
   patch :DATA
 
+  # fix GCC 5.0
+  patch do
+    url "https://raw.githubusercontent.com/Homebrew/patches/master/freeimage/gcc5.diff"
```

Comment:
you mean this `https://raw.githubusercontent.com/Homebrew/patches/f2b4fb191277d6124450c06a7664e9b24c91ffae/freeimage/gcc5.diff` ?